### PR TITLE
fix(fetch): copy the permisssion of the files during a helm fetch

### PR DESCRIPTION
**Pull request by kmala**

### CODE REVIEW

**Comment by mboersma** [comment]
@kmala this seems like a smart change, but I did a before / after test and I see the same file permissions anyway. Is there a bug that this fixes?

**Comment by kmala** [comment]
@mboersma  you can test using the https://github.com/deis/charts/pull/221 where there is script executing using the helm generate

**Comment by arschles** [comment]
Is there a unit test to write/modify in here to ensure that file perms get copied?

**Comment by mboersma** [comment]
> Is there a unit test...

There isn't anything that covers that package or `CopyDir()` yet.

File: util/helm_test.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package util
+
+import (
+	"testing"
```
Comment (mboersma) [inline util/helm_test.go]: Could you `gofmt` this file?

**Pull request merged**